manifests/repo/apt.pp

```diff
@@ -0,0 +1,92 @@
+#
+# == Class gluster::repo::apt
+#
+# enable the upstream Gluster Apt repo
+#
+# === Parameters
+#
+# version: the version to use when building the repo URL
+# repo_key_name: The repo signing key or fingerprint
+# repo_key_path: ignored
+# repo_key_source: ignored
+# priority: Apt pin priority to set for the Gluster repo
+#
+# Currently only released versions are supported.  If you want to use
+# QA releases or pre-releases, you'll need to edit line 54 below
+
+# === Examples
+#
+# Enable the Apt repo, and use the public key supplied
+#
+# class { gluster::repo::apt:
+#   repo_key_name => 'F7C73FCC930AC9F83B387A5613E01B7B3FE869A9',
+# }
+#
+# === Authors
+#
+# Drew Gibson <dgibson@rlsolutions.com>
+#
+# === Copyright
+#
+# Copyright 2015 RL Solutions, unless otherwise noted
+#
+class gluster::repo::apt (
+  $version         = $::gluster::params::version,
+  $repo_key_name   = $::gluster::params::repo_gpg_key_name,
+  $repo_key_source = $::gluster::params::repo_gpg_key_source,
+  $priority        = $::gluster::params::repo_priority,
+) {
+  include '::apt'
+
+  # basic sanity check
+  if $version == 'LATEST' {
+    $repo_ver = $version
```

Comment:
should we use $_version instead of $repo_ver? We already use $_version in the other class, so it woul be consistent naming scheme.